web/src/components/ThemedButton.jsx

```diff
@@ -0,0 +1,19 @@
+import { Button } from "antd";
+import { forwardRef } from "react";
+
+import "./ThemedButton.css";
+
+// eslint-disable-next-line react/prop-types
+export const ThemedButton = forwardRef(({ className, ...props }, ref) => {
```

Comment:
Use `Themed` prefix to customized component to express it is customized version

I prefer naming just `Input` instead of `ThemedInput`, but use `ThemedInput` to separate with antd components. If it's okay, I'll fix its name not to use prefix `Themed`.